docs/walkthrough_transcribe.md

```diff
@@ -2,22 +2,112 @@
 
-!!! warning
-    The section needs to be completed.
+Once the [segmentation of the images](walkthrough_segment.md) have been achieved and [some annotations](walkthrough_annotate.md) have been added, if needed, the next step in eScriptorium will be the text recognition of the documents.  
+
+This step can be done manually or automatically. Automatic text recognition is performed with a model (although you may still need some manual correction afterwards). Manual transcription can be done by typing the text directly in the interface, by copying and pasting text or by [importing XML transcriptions](walkthrough_import.md).
 
 ## Manual transcription
 
+For the manual transcription, two panels can be used:  
+
+1. "Transcription", which will display the transcription as it is structured in the images,  
+2. "Text", which will display the transcription as a plain succession of text lines, according to the order from the segmentation.
+
+![image: Transcription panel on eScriptorium (access button framed in red)](img/transcribe/transcription_panel.png "Transcription panel on eScriptorium (access button framed in red)")
+
+![image: Text panel on eScriptorium (access button framed in red)](img/transcribe/text_panel.png "Text panel on eScriptorium (access button framed in red)")
+
+### Transcription by hand
+
+When the transcription has to be done by hand, the best option is to use the "Transcription" panel.  
+
+To start transcribing, click on the corresponding zone. An input window is displayed. To record a transcription, press ++enter++: the transcription interface automatically displays the input field for the next segment.
+
+![image: Illustration of the transcription of the text by hand.](img/transcribe/by_hand.gif "Illustration of the transcription of the text by hand")
+
+<!-- I don't think this makes sense:
+!!! Note
+    If the transcription does not match the segmentation of the image, you have the possibility to modify the transcription in the lines through one of the two panels, until it matches completely.
+-->
+
 ## Automatic transcription
 
-## Virtual Keyboard
+You need a model to transcribe automatically, here are some ways to get one:  
+
+- by [training one with eScriptorium](walkthrough_train.md) or (via the command line) with [kraken](https://kraken.re),
+- by having one shared with you <!--either--> within [eScriptorium](walkthrough_collaborate.md#share-a-model),<!--or sent to you as a file through other means of communication,-->
+- by importing a model from a file. Such files can be retrieved on the [Zenodo community](https://zenodo.org/communities/ocr_models/) dedicated to Kraken models. Then can also be shared with you individually through other means of communication.
+
+### Predicting the transcription
+
+To learn how to apply the text recognition model and obtain a prediction, see the section ["Predict a segmentation or a transcription"](walkthrough_predict.md)
+
+### Editing the transcription
```

Comment:
There are elements here that should be at the beginning of the page (such as how to access the "Edition" panels)

As we discussed, I think "editing" and "transcribing by hands" are very similar as far as describing the feature is concerned. We could reorganize the beginning of the lesson like this : 

- preamble
- transcribe automatically -> send to predict
- use the "Transcribe" panel
- use the "Text" panel
- modify the transcription -> This section could be the occasion to precise that one might want to use the transcription versions to keep track of the modifications when they correct a transcription.

And in both "Transcribe" and "Text", mention that it is also possible use them to modify the transcription after it was done automatically.